### PR TITLE
Criteo: versioned Fastbid wrapper

### PR DESCRIPTION
**Pull request by bmwcmw**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Following last discussions https://github.com/prebid/Prebid.js/issues/5804 https://github.com/prebid/Prebid.js/issues/5239, we're offering a versioned wrapper of our fastbid script and allow users to specify fastBidVersion pointing to different source files. Therefore, the default version will be specified in the criteo adapter and be updated manually.

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer: 
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Review by msm0504 — APPROVED** [review]
@bmwcmw The changes look good to me. Let me know when this can be merged.

**Comment by bmwcmw** [comment]
Hi @msm0504 thanks, it could be part of the 5.0 release. I added a label, should I remove the "do not merge" now or is there anything else to do?

File: modules/criteoBidAdapter.js [hunk 1]
```diff
@@ -17,8 +17,11 @@ const storage = getStorageManager(GVLID);
 const LOG_PREFIX = 'Criteo: ';
 
 // Unminified source code can be found in: https://github.com/Prebid-org/prebid-js-external-js-criteo/blob/master/dist/prod.js
```
Comment (ncolletti) [inline modules/criteoBidAdapter.js:19]: Is there a plan to update this to include the un-minified publishertag code? This link is broken and the existing prebid-js-external repo is empty. If there isn't we could remove this line.

Comment (bmwcmw) [reply modules/criteoBidAdapter.js:19]: Hi @ncolletti, the link refers to a private repository shared to prebid so needs the access, it's kept here as a hint

**Comment by msm0504** [comment]
@bmwcmw There's now a branch for Prebid 5.0 changes. Can this PR be merged into that branch?

@ncolletti Can this PR be approved or are you waiting for more changes to be made?

**Comment by bmwcmw** [comment]
> @bmwcmw There's now a branch for Prebid 5.0 changes. Can this PR be merged into that branch?
> 
> @ncolletti Can this PR be approved or are you waiting for more changes to be made?

Hi @msm0504 @ncolletti it's ok for us to merge into 5.0 branch, I just removed the do not merge label.

**Pull request merged**